libraries/api/test/validate_test.js

```diff
@@ -129,6 +127,44 @@ suite(testing.suiteName(), function() {
     res.reply();
   });
 
+  builder.declare({
+    method: 'get',
+    route: '/test-double-json-send',
+    name: 'testDoubleJsonSend',
+    output: 'test-schema.yml',
+    category: 'API Library',
+    title: 'Test End-Point',
+    description: 'place to call to trigger a double send',
+  }, function(req, res) {
+    res.status(400).json({error: 'yep'});
+    res.status(200).reply({value: 1});
+  });
+
+  builder.declare({
+    method: 'get',
+    route: '/test-double-error-send',
+    name: 'testDoubleErrorSend',
+    output: 'test-schema.yml',
+    category: 'API Library',
+    title: 'Test End-Point',
+    description: 'place to call to trigger a double send',
+  }, function(req, res) {
+    res.reportError('InputError', 'uhoh', {});
+    res.status(200).reply({value: 1});
+  });
+
+  builder.declare({
+    method: 'get',
+    route: '/test-double-send',
+    name: 'testDoubleSend',
+    output: 'test-schema.yml',
+    category: 'API Library',
+    title: 'Test End-Point',
+    description: 'place to call to trigger a double send',
+  }, function(req, res) {
+    res.status(400).json('bad req');
+    res.status(200).reply({value: 1});
```

Comment:
what's the difference between this and `/test-double-json-send`?  Maybe this case should use `res.send(..)` instead?